README.md

```diff
@@ -312,6 +313,10 @@ Community-contributed DuckDB extensions, which can be installed via `INSTALL ⟨
 - [DuckDB Extension Radar](https://github.com/mehd-io/duckdb-extension-radar) - Repository that contains DuckDB extensions on GitHub. Refreshed daily.
 - [DuckDB extension weekly downloads](https://duckdb-ce-analysis.evidence.app/) - Statistics of weekly downloads for core extensions and community extensions. Refreshed daily.
 
+## Tutorials
+
+- [DBQuacks](https://dbquacks.com/tutorial/) - An interactive SQL tutorial powered by DuckDB.
```

Comment:
Getting a 404 there. I think the correct one is either `https://dbquacks.com` or `https://dbquacks.com/tutorial/1`